kani-driver/src/args.rs

```diff
@@ -361,6 +361,8 @@ pub enum UnstableFeatures {
     Stubbing,
     /// Generate a C-like file equivalent to input program used for debugging purpose.
     GenC,
+    /// Allow Kani to link against C code.
+    CFfi,
```

Comment:
I'm wondering if we should just have a generic FFI option (i.e., not restricted to C) as the gate for all FFI-related options. What do you think?

Reply:
Yeah, but does Kani work at all with any other type of FFI beyond C?

Reply:
Talked offline...